arduino-ide-extension/src/browser/widgets/cloud-sketchbook/cloud-sketchbook-contributions.ts

```diff
@@ -61,6 +64,14 @@ export namespace CloudSketchbookCommands {
     }
   }
 
+  export const SHOW_CLOUD_SKETCHBOOK_WIDGET = Command.toLocalizedCommand(
+    {
+      id: 'arduino-cloud-sketchbook--show-cloud-sketchbook-widget',
+      label: 'Show Cloud Sketchbook Widget',
+    },
+    'arduino/sketch/showCloudSketchbookWidget'
```

Comment:
Instead of having two separate commands, why does IDE2 not have one command, where the first ID is the tree widget ID (`local` or `cloud`) and the second arg is the `node` ID to reveal? You could also remove the dependency on the `EditorManager`.

Reply:
smart suggestion, I'll look into that!